File: TP/TP1/Code/5C.cpp
/* Afficher un triangle de hauteur N ; N étant demandé à USER */
#include <iostream>

int main()
{
  int a;
  
  std::cout << "Quelle est la hauteur de triangle ?" << std::endl;
  std::cin >> a;
  
  for (int i = 0; i < a; ++i) {
    for (int j = 0; j <= i; ++j) {
      std::cout << "*" << std::flush;
    }
    std::cout << std::endl;
  }
  
  return 0;
}

File: TP/TP1/Code/Python/5C.py
#!/usr/bin/python
# Afficher un triangle de hauteur N ; N étant demandé à USER
print('Donne moi la hauteur du triangle : ', end='')
foo = int(input())

for i in range(foo):
    for j in range(i + 1):
        print('*', end='')
    print()  # Ici on affiche un saut de ligne
